examples/nidcpower/measure-record.py

```diff
@@ -105,7 +105,7 @@ def check_for_warning(response, vi):
         nidcpower_types.SetAttributeViInt32Request(
             vi=vi,
             attribute_id=nidcpower_types.NiDCPowerAttribute.NIDCPOWER_ATTRIBUTE_MEASURE_RECORD_LENGTH,
-            attribute_value=RECORD_LENGTH,
+            attribute_value_raw=RECORD_LENGTH,
```

Comment:
Mypy was complaining about trying to put RECORD_LENGTH which is just the literal value 10 in `attribute_value` which expects and enum.